app/src/main/java/com/globant/calculatoronboardingkotlin/mvp/view/CalculatorView.kt

```diff
@@ -1,4 +1,21 @@
 package com.globant.calculatoronboardingkotlin.mvp.view
 
-class CalculatorView {
+import android.app.Activity
+import android.widget.TextView
+import com.globant.calculatoronboardingkotlin.mvp.contracts.CalculatorContracts
+import kotlinx.android.synthetic.main.activity_main.calculation_label
+import kotlinx.android.synthetic.main.activity_main.input_label
+
+class CalculatorView(activity: Activity) : ActivityView(activity), CalculatorContracts.View {
+
+    private val calculationLabel: TextView = activity.calculation_label
+    private val inputLabel: TextView = activity.input_label
+
+    override fun showResult(result: String?) {
```

Comment:
this parameter shouldn't be null?? use String only pls

Reply:
done, thanks